File: Iniciante/URI 1959.py
L = None
N = None
lista = None

def read_line():
  try:
    # read for Python 2.x
    return raw_input()
  except NameError:
    # read for Python 3.x
    return input()

lista = read_line().split(" ")
N = int((lista[0]))
L = int((lista[1]))
print(N * L)

File: Iniciante/URI 1959.cs
using System; 
 
class URI {
     
    static void Main(string[] args) { 
        string[] input = Console.ReadLine().Split(' ');
        long N = long.Parse(input[0]);
        long L = long.Parse(input[1]);
        long P = N * L;
        Console.WriteLine("{0}", P);
    }
    
}